test/hotspot/jtreg/testlibrary/jittester/Makefile

```
@@ -95,11 +97,15 @@ manifest:
 	@echo 'X-COMMENT: Main-Class will be added automatically by build' >> $(MANIFEST)
 	@echo 'Main-Class: jdk.test.lib.jittester.Automatic' >> $(MANIFEST)
 
+compile_asm: INIT
+	$(shell find $(ASM_SRC_DIR) -name '*.java' > $(CLASSES_DIR)/filelist_asm)
+	${JAVAC} -d $(CLASSES_DIR) @$(CLASSES_DIR)/filelist_asm
+
 compile_testlib: INIT
 	$(JAVAC) -XDignore.symbol.file --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED --add-exports=java.base/org.objectweb.asm=ALL-UNNAMED -Xlint $(TESTLIBRARY_SRC_FILES) -d $(CLASSES_DIR)
```

```suggestion
	$(JAVAC) -XDignore.symbol.file --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED -Xlint $(TESTLIBRARY_SRC_FILES) -d $(CLASSES_DIR)
```
As you did so below.